04/ht/calendar/calendar.js

```diff
@@ -0,0 +1,163 @@
+class Storage {
+  setData(key, value) {
+    Promise.resolve(localStorage.setItem(key, JSON.stringify(value)));
+  }
+  getData(key) {
+    return Promise.resolve(JSON.parse(localStorage.getItem(key)));
+  }
+}
+
+const dataNotes = {
+  id: "",
+  text: "",
+};
+
+const date = new Date();
+const storage = new Storage();
+
+function Calendar(options) {
+  this.id = idCheck(options);
+  createCalendar = createNewCalendar(this.id, options);
```

Comment:
зачем присваивать в поле объекта календарь? Оно ведь ни разу не вызывается.
```suggestion
  createNewCalendar(this.id, options);
```